src/SFA.DAS.AdminService.Web/Models/Error403ViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace SFA.DAS.AdminService.Web.Models
+{
+    public class Error403ViewModel
+    {
+        private readonly string _integrationUrlPart = string.Empty;
+
+        public Error403ViewModel(string environment)
+        {
+            if (!string.IsNullOrEmpty(environment) && !environment.Equals("prd", StringComparison.CurrentCultureIgnoreCase))
+            {
+                _integrationUrlPart = "test-";
+            }
+        }
+
+        /// <summary>
+        /// Gets DfESignIn Select service link.
+        /// </summary>
+        public string HelpPageLink => $"https://{_integrationUrlPart}services.signin.education.gov.uk/approvals/select-organisation?action=request-service";
+
+        /// <summary>
+        /// Gets or Sets UseDfESignIn config property.
+        /// </summary>
```

Comment:
This property is self-explanatory. It's best to leave comments like this out unless they substantially improve the clarity of the code